### PR TITLE
fix(actors): timer ticker bug

### PR DESCRIPTION
**Pull request by 1046102779**
The timer task needs to wait for the execution of the first dueTime's task to start the period's timer

### CODE REVIEW

**Comment by yaron2** [comment]
@vinayada1 can you review this?

File: pkg/actors/actors.go [hunk 1]
```diff
@@ -828,6 +828,10 @@ func (a *actorsRuntime) CreateReminder(ctx context.Context, req *CreateReminderR
 }
 
 func (a *actorsRuntime) CreateTimer(ctx context.Context, req *CreateTimerRequest) error {
+	var (
+		err             error
```
Comment (vinayada1) [inline pkg/actors/actors.go:832]: Could you please add a UT?

Comment (1046102779) [reply pkg/actors/actors.go:832]: it means Unit Test?

Comment (artursouza) [reply pkg/actors/actors.go:832]: Yes, unit test.

**Comment by yaron2** [comment]
@1046102779 we will merge this as is, thanks a lot for this fix. please find the time to add a unit test in a separate PR.

**Review by artursouza — APPROVED** [review]

File: pkg/actors/actors.go (hunk 1 above)
Comment (1046102779) [reply pkg/actors/actors.go:832]: [PR 2732](https://github.com/dapr/dapr/pull/2732)

**Pull request merged**